src/components/Book.js

```diff
@@ -1,20 +1,56 @@
 import React from "react";
 import "bootstrap/dist/css/bootstrap.min.css";
 import { useDispatch } from "react-redux";
+import { useEffect, useState } from 'react';
 import { DeleteBook } from "../redux/Books/book";
+import { CircularProgressbar } from "react-circular-progressbar"
+import 'react-circular-progressbar/dist/styles.css';
 
 const Book = (item) => {
-  const { id, title, author } = item;
+  const { id, title, author, category } = item;
   const dispatch = useDispatch();
   const deleteBook = () => {
     dispatch(DeleteBook(id));
   };
+  const [percentage, setPercentage] = useState(0);
+  useEffect(() => {
+    setTimeout(() => {
+      if (percentage < 100) {
+        setPercentage(percentage + 1);
+      }
+    }, 50);
+  }, [percentage]);
   return (
-    <div className="d-flex">
-      {" "}
-      <h2>{title}</h2>
-      <h2>{author}</h2>
-      <button className="btn btn-sm btn-danger" onClick={deleteBook}>Delete Book</button>
+    <div className="d-flex mt-3 container bg-white pt-3 pb-3">
+      <div className="d-grid w-50 ps-5">
+        {" "}
+        <p className="text-muted fw-bold">{category}</p>
+        <h2 className="mb-0">{title}</h2>
+        <p className="text-primary">{author}</p>
+        <div className="d-flex gap-3">
+          <a href="##" className="text-primary nav-link">Comments</a>
+          <p className="text-muted">|</p>
+          <a href="##" className="text-primary nav-link" onClick={deleteBook}>Remove</a>
+          <p className="text-muted">|</p>
+          <a href="##" className="text-primary nav-link">Edit</a>
+        </div>
+      </div>
+      <div className="w-25 d-flex mt-4">
+        {" "}
+        <div style={{ width: 75, marginLeft: 20 }}>
+          <CircularProgressbar value={percentage} />
+        </div>
+        <div className="mt-3 ms-2">
+          <p className="mb-0">{`${percentage}%`}</p>
+          <p className="text-muted">Completed</p>
+        </div>
+        <div class="Line-2"></div>
+      </div>
```

Comment:
- [ ] You should stick to the design as much as possible like Font, color, and spacing. There is a logo at the top of the page. please implement it. Please use the same font and color that is in the design.
### Your design
![your](https://user-images.githubusercontent.com/81727988/202830599-27355096-0291-4db3-9416-07b308153519.png)
### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/202830605-fb1b7266-b538-4e85-bdf2-58cb117b0fa2.png)